main_test.go

```diff
@@ -31,7 +32,7 @@ func (f *ComponentTest) InitializeScenario(ctx *godog.ScenarioContext) {
 
 	ctx.AfterScenario(func(*godog.Scenario, error) {
 		if err = component.Close(); err != nil {
-			log.Event(nil, "error closing identity component", log.Error(err), log.WARN)
+			log.Warn(context.Background(), "error closing identity component", log.FormatErrors([]error{err}))
```

Comment:
What is the reason for using:
log.FormatErrors([]error{err}
instead of just:
err
?

Reply:
I think it's an error caused by the automatic process I'm using.

I'll simplify it.

Reply:
The func signature is Warn(ctx context.Context, event string, opts ...option) and doesn't contain an error type.

Reply:
Ok.Thansk.